docs/_articles/en/index.md

```diff
@@ -4,16 +4,28 @@ lang: en
 permalink: /
 ---
 
-This extension pack includes tools for developing on the Salesforce platform in the lightweight, extensible VS Code editor. These tools provide features for working with development orgs (scratch orgs, sandboxes, and DE orgs), Apex, Aura components, and Visualforce.
+## One Set of Extensions, Two Ways to Code
+
+You can install the Salesforce Extensions in VS Code on the desktop or access them via Code Builder in your browser.
+
+The Salesforce Extension pack includes tools for developing on the Salesforce platform in the lightweight, extensible VS Code editor. These tools provide features for working with development orgs (scratch orgs, sandboxes, and DE orgs), Apex, Aura components, and Visualforce.
+
+### Salesforce Extensions for Desktop
+
+Want to develop locally? Already using VS Code? Install Salesforce Extensions and Salesforce CLI to develop quickly and more productively on Salesforce platform.
 
 <span>Download using the links below, or read the [installation instructions](./en/getting-started/install).<span><br/><a class="slds-button slds-button_neutral landing__header-cta slds-m-vertical--x-large" href="https://code.visualstudio.com">Download Visual Studio Code</a><a class="slds-button slds-button_brand landing__header-cta slds-m-vertical--x-large" href="https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode">Install the Salesforce Extensions</a>
 
 ![GIF showing Apex code completion, pushing source to a scratch org, and running Apex tests](./images/overview.gif)
 
+### Code Builder (Beta)
```

Comment:
@ekapner - below in the videos, can you also update it to say "Code Builder (beta)"?  It currently says "Code Builder (pilot)".

Reply:
Actually, @smaddox-sf, out of an abundance of caution, I'm going to keep both references as pilot for now. Until we officially launch to avoid confusion. I'll open a doc work item to change any instances here to pilot once it's official.